docs/user-docs/custom-css.md

```diff
@@ -87,8 +87,16 @@ For example, if your column name is `text (name)`, chaise will rewrite this as `
 
 ### Examples:
 
+- Added a class to with deal column spacing in Firefox browser on Mac
```

Comment:
It might be better to move this to the top section. https://github.com/informatics-isi-edu/chaise/blob/master/docs/user-docs/custom-css.md#changing-chaise-table-styles

Can you change the title to `Changing chaise styles` and modify the first paragraph after that heading to:
 - With the current HTML structure, it is possible to apply different styles to different parts of chaise and to further style the chaise tables based on model information.

Make sure to use the classes you added since this has `.addMacFirefoxClass` in your example.

Reply:
Corrected this